Cargo.lock

```diff
@@ -397,9 +380,9 @@ dependencies = [
 
 [[package]]
 name = "block-buffer"
-version = "0.10.0"
+version = "0.10.2"
```

Comment:
Please revert to 0.10.0

Reply:
This is required by digest v0.10.3 which is required by solana-sdk v1.11.0